src/testhost.x86/DefaultEngineInvoker.cs

```diff
@@ -100,7 +101,16 @@ public void Invoke(IDictionary<string, string?> argsDictionary)
                 .GetTypeInfo()
                 .Assembly
                 .GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion;
-            EqtTrace.Verbose($"Version: {version}");
+            EqtTrace.Verbose($"Version: {version} Current process architecture: {_processHelper.GetCurrentProcessArchitecture()}");
+#if NETCOREAPP2_0_OR_GREATER || NETFRAMEWORK
+            // https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.location?view=net-6.0#remarks
+            // In .NET 5 and later versions, for bundled assemblies, the value returned is an empty string.
+            string objectTypeLocation = typeof(object).Assembly.Location;
+            if (!string.IsNullOrEmpty(objectTypeLocation))
```

Comment:
Can you use the helper here?

```suggestion
            if (!objectTypeLocation.IsNullOrEmpty())
```